### PR TITLE
Finish Edit Profile scene with updateProfile methods

### PR DESCRIPTION
**Pull request by soniasophia**

### CODE REVIEW

File: js/scenes/AccountSettings/AccountSettings.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  Image,
+  TextInput,
+  ScrollView,
+  TouchableOpacity
+} from 'react-native';
+import SurveyButton from '../../components/SurveyButton/';
+import { styles } from './styles';
+
+const AccountSettings = ({ handleImageUpload, imageUrl, handleEmail, emailField, handlePassword, passwordField, handleFullname, fullnameField }) => {
+  return (
```
Comment (codedavinci) [inline js/scenes/AccountSettings/AccountSettings.js:14]: If there's no variable declaration, you don't need a `return` statement.

File: js/scenes/AccountSettings/AccountSettingsContainer.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  updateEmailField, 
+  updatePasswordField,
+  updateFullnameField,
+  updateImageUrlField
+} from '../../redux/modules/user-forms';
+
+
+class AccountSettingsContainer extends Component {
+
+  static route = {
```
Comment (codedavinci) [inline js/scenes/AccountSettings/AccountSettingsContainer.js:17]: if there's no `state` or life cycle methods, choose for a `functional component`

File: js/scenes/AccountSettings/AccountSettingsContainer.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+  render() {
+    return (
+      <AccountSettings 
+        emailField={this.props.emailField}
+        passwordField={this.props.passwordField}
+        fullnameField={this.props.fullnameField}
+        imageUrl={this.props.imageUrl}
+
+        handleEmail={(e) => {
```
Comment (codedavinci) [inline js/scenes/AccountSettings/AccountSettingsContainer.js:54]: functions with one line of code don't need closing scope.

File: js/scenes/Profile/Profile.js [hunk 4]
```diff
@@ -1,16 +1,173 @@
...
-    <View>
-      <Text>This is the profile scene.</Text>
-    </View>
+    <ScrollView style={styles.container}>
+      <View>
+        <View style={styles.imageContainer}>
+          <TouchableOpacity style={styles.imageContainer} onPress={() => handleImageUpload()}>
+            <Image
+              style={styles.image}
+              source={{ uri: (imageUrl ? imageUrl || user.imageUrl : 'https://image.ibb.co/moG48k/account_icon.png') }}
```
Comment (codedavinci) [inline js/scenes/Profile/Profile.js]: If this comes `undefined`, it will break.

File: js/scenes/Profile/Profile.js [hunk 5]
```diff
@@ -1,16 +1,173 @@
...
+            />
+            <Image
+              style={styles.pencil}
+              source={require('../../assets/icons/pencil.png')}
```
Comment (codedavinci) [inline js/scenes/Profile/Profile.js]: `import ` rather than `require`.

File: js/scenes/Profile/Profile.js [hunk 6]
```diff
@@ -1,16 +1,173 @@
...
+            />
+          </TouchableOpacity>
+        </View>
+        <Text style={styles.headings}>Personal Bio</Text>
+        <View style={styles.fieldsContainer}>
+          <TextInput
+            style={styles.inputBig}
+            onChangeText={(text) => handleBio(text)}
+            value={
+              bioField ?
```
Comment (codedavinci) [inline js/scenes/Profile/Profile.js]: If it's a 50/50 logic use ` or = ||`  or `and = &&`

File: js/scenes/Profile/Profile.js [hunk 7]
```diff
@@ -1,16 +1,173 @@
...
+            maxLength={100}
+            placeholder="Goal #2"
+          />
+        </View>
+        <View style={styles.fieldsContainer}>
+          <TextInput
+            style={styles.inputBig}
+            onChangeText={(text) => handleGoals(text, "goalThree")}
+            value={
+              goalsField.goalThree ?
```
Comment (codedavinci) [inline js/scenes/Profile/Profile.js]: If it's a 50/50 logic use ` or = ||`  or `and = &&`

File: js/scenes/Profile/ProfileContainer.js [hunk 8]
```diff
@@ -1,13 +1,117 @@
...
+  updateImageUrlField
+} from '../../redux/modules/user-forms';
+
+import { customFieldsUpdater } from '../../config/helpers';
+import { auth } from '../../config/firebase';
 
 class ProfileContainer extends Component {
 
+  static route = {
+    navigationBar: {
```
Comment (codedavinci) [inline js/scenes/Profile/ProfileContainer.js]: If the component doesn't hold `state` or life cycle methods, opt for function components

**Review by codedavinci — COMMENTED** [review]
To be adjusted

File: js/scenes/AccountSettings/AccountSettings.js (hunk 1 above)
Comment (soniasophia) [reply js/scenes/AccountSettings/AccountSettings.js:14]: Please ignore the AccountSettings components as it's still being worked on (included in the PR by accident during the squashing of commits).

File: js/scenes/AccountSettings/AccountSettingsContainer.js (hunk 2 above)
Comment (soniasophia) [reply js/scenes/AccountSettings/AccountSettingsContainer.js:17]: Please ignore the AccountSettings components as it's still being worked on (included in the PR by accident during the squashing of commits).

File: js/scenes/AccountSettings/AccountSettingsContainer.js (hunk 3 above)
Comment (soniasophia) [reply js/scenes/AccountSettings/AccountSettingsContainer.js:54]: Please ignore the AccountSettings components as it's still being worked on (included in the PR by accident during the squashing of commits).

**Review by codedavinci — APPROVED** [review]
LGTM

**Pull request merged**